src/content/en/tools/chrome-devtools/network-performance/issues.md

```diff
@@ -0,0 +1,123 @@
+project_path: /web/_project.yaml
+book_path: /web/tools/_book.yaml
+description: Learn how to detect network issues in the Network panel of Chrome DevTools.
+
+{# wf_published_on: 2016-12-28 #}
+{# wf_updated_on: 2016-12-28 #}
+
+<style>
+figcaption {
+  text-align: center;
+}
+</style>
+
+# Network Issues Guide {: .page-title }
```

Comment:
I took some of the content from [Understanding Resource Timing](https://developers.google.com/web/tools/chrome-devtools/network-performance/understanding-resource-timing) and moved to its own doc. The idea is that once you understand the basics of using the network panel, then all you really need is a reference of request activity patterns that indicate issues or optimization opportunities.